config/locales/en.yml

```diff
@@ -404,6 +404,8 @@ en:
     beta: Beta
     beta_message_html: This is a new service - your <a href="%{user_survey_link}" target="_blank">feedback</a> will help us improve it
     beta_message_short: This is a new service
+    beta_message_pilot_heading: This page is no longer active as the pilot phase is complete.
+    beta_message_pilot_body_html: Thank you for taking part. </br> <a href="https://check-your-client-qualifies-for-legal-aid.service.gov.uk" target="_blank">Check if your client qualifies for legal aid</a> is now live and available to all users.
```

Comment:
Also, rather than hard-code this URL here, I'd use `ENV["PRIMARY_HOST"]` passed in as a variable

Reply:
What it the syntax? I've arrived at this:

```
public_beta_link: <%= ENV["PRIMARY_HOST"] %>
beta_message_pilot_body_html: Thank you for taking part. </br> <a href="%{public_beta_link}" target="_blank">Check if your client qualifies for legal aid</a> is now live and available to all users.

```